src/test/java/io/debezium/connector/yugabytedb/YugabyteDBTabletSplitTest.java

```diff
@@ -107,12 +115,7 @@ public void shouldConsumeDataAfterTabletSplit() throws Exception {
     ybClient.splitTablet(tablets.iterator().next());
 
     // Wait till there are 2 tablets for the table.
-    Awaitility.await()
-      .pollDelay(Duration.ofSeconds(2))
-      .atMost(Duration.ofSeconds(20))
-      .until(() -> {
-        return ybClient.getTabletUUIDs(table).size() == 2;
-      });
+    waitForTablets(ybClient, table, 2);
```

Comment:
Is this the only place the function will be used? If yes, I suggest not wrapping in a fn.

Reply:
There can be more test additions in the same class, all of them are gonna require the code block this function wraps.